plugins/rollbar/src/api/RollbarMockClient.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+import { RollbarApi } from './RollbarApi';
+import {
+  RollbarItemsResponse,
+  RollbarProject,
+  RollbarTopActiveItem,
+} from './types';
+
+export class RollbarMockClient implements RollbarApi {
```

Comment:
Not sure if the `example-app` should use the mock versus the real `RollbarClient`.

Reply:
Stick to the current way and let's see what makes sense. Awesome that you added it either way 👍